scalyr_agent/configuration.py

```diff
@@ -3640,6 +3714,11 @@ def __perform_substitutions(self, source_config):
                 object_value=source_config, substitutions=substitutions
             )
 
+    def __getstate__(self):
```

Comment:
the configuration object can not be serialized due to the logger object, so it is impossible to pickle them and send to the multiprocess workers.
Another solution is to read the configuration one more time on each worker,